src/modes/direct_select.js

```diff
@@ -128,7 +134,10 @@ DirectSelect.onSetup = function(opts) {
     dragMoveLocation: opts.startPos || null,
     dragMoving: false,
     canDragMove: false,
-    selectedCoordPaths: opts.coordPath ? [opts.coordPath] : []
+    selectedCoordPaths: opts.coordPath ? [opts.coordPath] : [],
+    draggablePoint: opts.draggablePoint !== undefined ? opts.draggablePoint : true,
+    draggableLine: opts.draggableLine !== undefined ? opts.draggableLine : true,
+    draggablePolygon: opts.draggablePolygon !== undefined ? opts.draggablePolygon : true
```

Comment:
A purely syntax alternative would be to use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing):

`draggablePoint: opts.draggablePoint ?? true,`

Approving anyway, even if you decide not to change it.